stories/General/ButtonStory.tsx

```diff
@@ -44,6 +44,21 @@ const buttonProps = [
     require: 'no',
     description: 'Sets Button to small version.',
   },
+  {
+    name: 'tag',
+    type: 'string',
```

Comment:
Maybe `React.ElementType` is more suitable

Reply:
Changed it.